### PR TITLE
Diagrams and "looking ahead" sections

### LINKED ISSUE #461: Chapter diagrams feedback
Diagrams are in DIME Analytics\d4di\Chapter diagrams

### PR DESCRIPTION
**Pull request by luizaandrade**
Adding diagrams resolves #461.
I went back and forth between having them at the end of the chapter or in the beginning, but it seemed to me that it could go well with the final chapter paragraphs mentioned in #482, so I left them at the end.
Let me know what you all think.

### CODE REVIEW

**Comment by bbdaniels** [comment]
These are so awesome! Very minor things left:

- [ ] Overall
  - [ ] I think the green and the blue in Design and Acquisition are really close but I'm OK with it. Very calming color palette choices 😌
- [ ] "Intro"
  - [ ] Change file name to "Introduction" 
- [ ] "Acquisition"
  - [ ] Center text on "Manuals and protocols" 
- [ ] "Cleaning"
  - [ ] Typo "Recodie" should be "Recode"
- [ ] "Publication"
  - [ ] Outer border color and box fill does not match header color, it should be the color from "Conclusion"
  - [ ] Change "Manuscript" to "Final output"?
  - [ ] Not sure I love the overlapping icons but happy to hear what everyone things
- [ ] "Conclusion"
  - [ ] Should "Data Work Map" just be "Data map" (watch capitalization)
  - [ ] Should "Cleaned data" be "Clean data"

**Review by bbdaniels — APPROVED** [review]

File: chapters/0-introduction.tex [hunk 1]
```diff
@@ -156,15 +156,12 @@ \section{How to read this book}
...
-\begin{fullwidth}
-\begin{figure}
-	\centering
-	\includegraphics[width=1.25\linewidth]{diagrams/Intro}
-	\caption{}
-	\label{fig:intro}
-\end{figure}
-\end{fullwidth}
-
+Figure \ref{fig:intro} shows a summary of the hands-on data tasks
```
Comment (kbjarkefur) [inline chapters/0-introduction.tex:159]: The ref number does not show up for me:

image

Comment (bbdaniels) [reply chapters/0-introduction.tex:159]: I would cut figure numbering and refs entirely, just call them by name?

File: chapters/0-introduction.tex [hunk 2]
```diff
@@ -156,15 +156,12 @@ \section{How to read this book}
...
-	\caption{}
-	\label{fig:intro}
-\end{figure}
-\end{fullwidth}
-
+Figure \ref{fig:intro} shows a summary of the hands-on data tasks 
+comprising  research data work.
+These are described in chapters 3 to 7.
+Chapters 1 and 2, on the other hand,
+will contextualize why these each of them are important,
```
Comment (kbjarkefur) [inline chapters/0-introduction.tex:163]: I do not understand this part of the sentence

File: chapters/0-introduction.tex [hunk 3]
```diff
@@ -156,15 +156,12 @@ \section{How to read this book}
...
-	\includegraphics[width=1.25\linewidth]{diagrams/Intro}
-	\caption{}
-	\label{fig:intro}
-\end{figure}
-\end{fullwidth}
-
+Figure \ref{fig:intro} shows a summary of the hands-on data tasks 
+comprising  research data work.
+These are described in chapters 3 to 7.
+Chapters 1 and 2, on the other hand,
```
Comment (kbjarkefur) [inline chapters/0-introduction.tex:162]: Can we put this before we talk about what each chapter is about? It feels a bit weird that we give a high level over view when we have already talked about the chapters in some details.

File: chapters/4-acquisition.tex [hunk 4]
```diff
@@ -807,6 +798,23 @@ \subsection{Backing up original data}
 If you remain lucky, you will never have to access your ``master'' or ``golden master'' copies --
 you just want to know it is there, safe, if you one day end up needing it.
 
-Once the raw data is securely stored and backed up,
-it can be transformed into your final analysis dataset
-through the steps described in the next chapter.
+\section{Looking ahead}
+
+Once the raw data is completely transferred, securely stored and backed up,
+the data acquisition stage, summarized in figure \ref{fig:acquisition}, is complete.
```
Comment (kbjarkefur) [inline chapters/4-acquisition.tex:804]:
```suggestion
then the data acquisition stage, summarized in figure \ref{fig:acquisition}, is complete.
```
I had to read this sentence a couple of times. I didnt get it until I added a "then" in my mind

**Comment by kbjarkefur** [comment]
First of all, these diagrams is sooo good!

**General comments**

- [ ] I think we should use only one icon for documentation, and not the "multiple documentation icon" used, for example, in he Acquisition diagram for _Manuals and Protocols_. Many of the documentation that has the "single documentation icon" will likely also have multiple components. I do not think the point of indicating what is more than one thing justifies not making it as simple as possible.
- [ ] I think it should always be "disclosure risk" and not "risk of disclosure". Disclosure means making something known. We want to make things known when we publish things. But when we disclose things there is a risk that we disclose things we do not want to make known. So there is a risk associated with disclosure, ie. "disclosure risk", but some disclosure is often intentional and therefore  we do not worry about "risk of disclosure". Both in intro.png and publication.png

**Cleaning.png**

- [ ] The end point is called "clean data" but it is called "cleaned data" in the conclusion, processing and publication diagram. We should be consistent.

**Analysis.png**

- [ ] I am not fan of the curly bracket as it brakes the style. Can we do three regular arrows from the three boxes "Treat missing...". "Integrate data..." and "Create analysis..."
- [ ] If we should be consistent then there should be an arrow from "Analysis data" and "Analysis code" to "publication"
- [ ] Polish the results. doesnt it sound as if we try to make the resutls better than they are? Isnt "formatting results" better?

**publication.png**

- [ ] Why do we make a difference between "License and terms of use" of replication package and "terms of use" for data publication package? Why no license for data?

**intro.png**

- [ ] Why is only non-original data have an arrow back to sampling? I do not think we need this arrow, but if we should keep it, isnt it more fitting that the original data branch point to sampling?

**conclusion** 

- [ ] we do not mention original data in any other diagram

**Comment by luizaandrade** [comment]
> **Analysis.png**
> 
> * [ ]  I am not fan of the curly bracket as it brakes the style. Can we do three regular arrows from the three boxes "Treat missing...". "Integrate data..." and "Create analysis..."
> * [ ]  If we should be consistent then there should be an arrow from "Analysis data" and "Analysis code" to "publication"
> * [ ]  Polish the results. doesnt it sound as if we try to make the resutls better than they are? Isnt "formatting results" better?
> 
> **publication.png**
> * [ ]  Why do we make a difference between "License and terms of use" of replication package and "terms of use" for data publication package? Why no license for data?

We typically release terms of use for the data on the microdata catalog, and add a license to the code on GitHub. But if that doesn't seem right, I can change it.

File: chapters/0-introduction.tex (hunk 2 above)
Comment (luizaandrade) [reply chapters/0-introduction.tex:163]:
```suggestion
will contextualize why each of them are important,
```

does this help?

File: chapters/0-introduction.tex (hunk 1 above)
Comment (luizaandrade) [reply chapters/0-introduction.tex:159]: done

File: chapters/0-introduction.tex (hunk 3 above)
Comment (luizaandrade) [reply chapters/0-introduction.tex:163]: Sure. Feel free to move it wherever it seems like a better fit, or to remove it altogether.

Comment (kbjarkefur) [reply chapters/0-introduction.tex]: @luizaandrade , I did this in 767ef1359d8c09a62ba0c2463430f5f100b81d10. Is this ok to you?

**Comment by bbdaniels** [comment]
@kbjarkefur are you done working here?

**Comment by kbjarkefur** [comment]
Yes, but someone should review my last commit

**Comment by bbdaniels** [comment]
Done, although I thought the captions didn't fit and removed them. The intro one is fine, let's see what other people say before fiddling any more with placement. Merging!

**Pull request merged**